README.md

```
@@ -1,26 +1,44 @@
-To produce mapping from a python program:
-```bash
-python3 instrumenter.py --mode parser <source_path>
+|   | Type        | Convention           | Description                      | Example                          |
+|---|-------------|----------------------|----------------------------------|----------------------------------|
+| A | Behavior    | b_`<behavior>`         | The behavior being exhibited.    | def b_acceptName():              |
+| B | Argument    | p_arg_`<argument>`     | The arguments into the behavior. | p_arg_name = name                |
+| C | Participant | p_pre_`<participant>`  | The pre behavior world state.    | p_pre_basket = basket            |
+| D | Participant | p_post_`<participant>` | The post behavior world state.   | p_post_firstLetter = firstLetter |
+
+Example using accept name behavior (argument as input from environment):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)